AirlineReservation/Controllers/AirlineController.cs

```diff
@@ -20,5 +21,56 @@ public async Task<IActionResult> GetAllAirlines()
             var airlines = await _airlineService.GetAllAirlines();
             return Ok(airlines);
         }
+
+        [HttpGet("/{airlineId}")]
+        public async Task<IActionResult> GetAirlineById(string airlineId)
+        {
+            var airline = await _airlineService.GetAirlineById(airlineId);
+            return airline == null ? NotFound() : Ok(airline);
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> CreateAirline(AirlineDto airlineDto)
+        {
+            var createAirline = await _airlineService.CreateAirline(airlineDto);
+            return createAirline == null ? NotFound() : CreatedAtAction(nameof(CreateAirline),
+            new { id = createAirline.AirlineId }, createAirline);
+        }
+
+        [HttpPut("/{airlineId}")]
```

Comment:
use route contraints 
the slash ahead is not required

Reply:
used routes